lib/plugins.py

```diff
@@ -605,10 +608,7 @@ def close(self):
         self._hooks_i_registered.clear()  # just to kill strong refs to self ASAP, for GC
         # remove registered daemon commands
         for cmdname in self._daemon_commands:
-            try:
-                del self.parent.daemon_commands[cmdname]
-            except KeyError:
-                pass # again, shouldn't happen
+            self.parent.daemon_commands.pop(cmdname, None)
         self._daemon_commands = ()
```

Comment:
Now I hate to be a stickler.. and this was my insistence originally to even clear it.. but maybe do: `self._daemon_commands.clear()` here so it keeps type `set`?